README.md

```diff
@@ -45,15 +45,18 @@ As go plugins can be finicky to correctly compile and install, you may want to c
 # We use go modules for everything.
 > export GO111MODULE=on
 
+# (Optionally) clean go cache
+go clean -modcache; go clean -cache
+
 # Clone kubo.
 > git clone https://github.com/ipfs/kubo
 > cd kubo
 
 # Pull in the datastore plugin (you can specify a version other than latest if you'd like).
-> go get github.com/ipfs/go-ds-s3/plugin@latest
+> go get github.com/ipfs/go-ds-s3/go-ds-s3-plugin@latest
 
 # Add the plugin to the preload list.
-> echo -en "\ns3ds github.com/ipfs/go-ds-s3/plugin 0" >> plugin/loader/preload_list
+> echo -en "\ns3ds github.com/ipfs/go-ds-s3/go-ds-s3-plugin 0" >> plugin/loader/preload_list
```

Comment:
Same comment about import path here.

Reply:
Removing "/go-ds-s3-plugin" doesn't work here. Build error: `plugin/loader/preload.go:28:21: undefined: plugins3ds.Plugins`
I assume GO needs the specific path.

Reply:
Setting it to "github.com/ipfs/go-ds-s3/plugin" doesn't work as well. 
-> I only manage to get it work with "github.com/ipfs/go-ds-s3/go-ds-s3-plugin".

Reply:
Your PR hasn't been merged nor released, so it can't be picked up automatically by go mod, did you tried adding a temporary `replace github.com/ipfs/go-ds-s3 => ../go-ds-s3` to your Kubo `go.mod` file ? :slightly_smiling_face:

Reply:
Yes I'm aware of that. I'm testing on a [fork](https://github.com/chixodo-xyz/go-ds-s3) so I can mess around with master branch. 
Temporary adding `replace github.com/ipfs/go-ds-s3 => ../go-ds-s3` doesn't change the behavior described above.

Reply:
I merge and this can be fixed in upcoming PR if still doesn't work.